WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -559,10 +564,11 @@ def __doOSG ( self ):
     """ Treat the OSG case """
 
     osgDir = ''
+    vo = self.pp.releaseProject.replace( 'DIRAC', '' ).upper()
```

Comment:
This is certainly not a correct way to evaluate the VO. The best way to do that is to get it based on the pilot proxy DIRAC group